types/module/server/testutil.go

```diff
@@ -77,33 +77,45 @@ func (ff FixtureFactory) Setup() testutil.Fixture {
 	baseApp := ff.baseApp
 	baseApp.MsgServiceRouter().SetInterfaceRegistry(registry)
 	baseApp.GRPCQueryRouter().SetInterfaceRegistry(registry)
-	mm := NewManager(baseApp, cdc)
-	err := mm.RegisterModules(ff.modules)
-	require.NoError(ff.t, err)
-	err = mm.CompleteInitialization()
-	require.NoError(ff.t, err)
-	err = baseApp.LoadLatestVersion()
+	mm := sdkmodules.NewManager(ff.modules...)
+	cfg := sdkmodules.NewConfigurator(cdc, baseApp.MsgServiceRouter(), baseApp.GRPCQueryRouter())
+	for _, module := range mm.Modules {
+		module.RegisterInterfaces(ff.cdc.InterfaceRegistry())
+		module.RegisterServices(cfg)
+	}
+
+	// TODO(Tyler): ???
+	//mm := NewManager(baseApp, cdc)
+	//err := mm.RegisterModules(ff.modules)
+	//require.NoError(ff.t, err)
+	//err = mm.CompleteInitialization()
+	//require.NoError(ff.t, err)
```

Comment:
Not sure about this. Looking into it.

Reply:
this was for the custom module code, just deleted all of it